src/components/banner/Banner.tsx

```diff
@@ -0,0 +1,69 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+import * as _ from 'underscore';
+
+import {Svg} from '../svg/Svg';
+import * as styles from './styles/Banner.scss';
+
+export interface BannerProps {
+    name: string;
+    nameSubtitle?: string;
+    messageTitle?: string;
+    messageDescription?: React.ReactNode;

```

Comment:
use children instead

Reply:
I feel like it's unclear what the children should be. I think it's easier to know what goes where if there is a prop for it.

Reply:
I mean there are three props receiving a React.ReactNode.

Reply:
To me it is pretty clear that the children should be the main content in the middle of the banner. With `topRightInfos` and `bottomRightInfos` as props, it gets even clearer